src/test/java/com/bringframework/component/TestService.java

```diff
@@ -0,0 +1,9 @@
+package com.bringframework.component;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
```

Comment:
Are these annotations needed ?

Reply:
As this class is ampty, and used just for test, maybe not. Removed